modules/jenkins-fortify-slave/removeNode.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Is this script just for debugging? Looks like this is covered in the terraform config now. If so, can we remove this from the repo?